pkg/data/components/OTelDebugExporter.yaml

```diff
@@ -1,5 +1,6 @@
-name: otel_debug_exporter
 kind: OTelDebugExporter
+name: otel_debug_exporter
```

Comment:
The name on some of the other components use some camel-casing e.g. `EMA_Throughput_Sampler`. Is there a way we can make these all consistent? Also happy to do as a follow-up once it's decided. Main idea of consistent casing style would be because it shows up in the UI.

Reply:
I decided that we can use actually natural naming including spaces, so I renamed them all to look like `OTel Receiver`